notebooks/ranking_signals.ipynb

```diff
@@ -0,0 +1,1005 @@
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div style=\"display: flex; align-items: center;\"><div style=\"flex: 1;\"><img src=\"https://d11p8vtjlacpl4.cloudfront.net/kaggle-hm-images/055/0552852035.jpg\" height=\"120\" width=\"85\"></div><div style=\"flex: 2;\"><p>ID: 552852035</p><p>Product Name: SALT Regular tee (1)</p><p>Menswear &raquo; Garment Upper body &raquo; T-shirt</p><p>Color: Dark Green - 93</p><p>Rating: 1</div></div>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ranking_expr = 'uncalibrated_relevance * if(object.index_group_name == \"Menswear\", 1.02, 1)'\n",
+    "\n",
+    "results = index.search(\"t-shirt\", object_fields=\"*\", limit=20, ranking_expr=ranking_expr)\n",
+    "display_results(results)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's take a closer look at the ranking expression:\n",
+    "\n",
+    "```\n",
+    "uncalibrated_relevance * if(object.index_group_name == \"Menswear\", 1.02, 1)\n",
+    "```\n",
+    "\n",
+    "`uncalibrated_relevance` is a primitive in Ranking Signals that represents the raw relevance score computed by the index before any ranking expression is applied. A ranking expression must return a new score.\n",
+    "\n",
+    "The `if(object.index_group_name == \"Menswear\", 1.02, 1)` part of the expression is saying if the `index_group_name` equals `Menswear`, return `1.02`, else return `1`.\n",
+    "\n",
+    "Finally, we are multiplying the result of the `if` statement and the `uncalibrated_relevance`. \n",
+    "\n",
+    "To see more examples and a full list of the available functions in Ranking Expressions, see our [docs](https://www.objective.inc/docs/apis/search/text-search#ranking-signals)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Add Another Signal\n",
+    "\n",
+    "Our objects also have a `rating` attribute with values from 1 to 5. Let's extend our ranking signal so that we give a 2% boost to any objects having a rating equal to or greater than 4. To accomplis this, we'll follow the same pattern as our `Menswear` boost but we will add it to the existing expression by multiplying them together. This means that any object that is in the `Menswear` category with a rating of over 4 or more will receive a `4%` boost."
+   ]
```

Comment:
we're going to denote `Menswear` using a code snippet we should also do the same for "rating" in the last sentence.